ppr-ui/src/interfaces/state-interface.ts

```diff
@@ -1,6 +1,7 @@
-import { StateModelIF } from '@/interfaces'
+import { AccountModelIF, StateModelIF } from '@/interfaces'
 
 // State model example
 export interface StateIF {
   stateModel: StateModelIF
+  account: AccountModelIF
```

Comment:
I don't this merits an entirely separate model, there is account related items already in our state.
You can include this property in our current stateModel, just perhaps a name which describes these properties more concisely.
Could these properties be included in `accountInformation`? How do they differ?

Reply:
This is a different state model that is set by `SbcSign` common component (set during sign on). It is similar and some props are the same (within our app's state), but it set earlier and we have access during Dashboard load.